html/changelogs/mattatlas-aughmyartery.yml

```diff
@@ -0,0 +1,45 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: MattAtlas
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscdel: "Internal bleeding has been removed."
+  - rscadd: "It has been replaced with the much more impactful arterial bleeding. They will cause blood squirts in random directions. This is healed via fixovein in surgery or bicaridine overdose. Bandages will stave off some of the bleeding."
+  - tweak: "Cutting people's throats now cuts their cartoid artery as well."
+  - rscadd: "Added tendon cutting. This will make your limb unusable and is applied on random chance based on damage when someone is damaged."
```

Comment:
```suggestion
  - rscadd: "Added tendon cutting. This will make your limb unusable and is applied on random chance based on damage when someone is damaged. Repaired with fixovein in surgery."
```